RELEASE/scripts/sl_ascend.ash

```diff
@@ -56,6 +56,8 @@ import <sl_ascend/sl_list.ash>
 import <sl_ascend/sl_zlib.ash>
 import <sl_ascend/sl_zone.ash>
 
+// Variable used to force the tavern
+boolean forceTavern = false;
```

Comment:
Use a preference, not a global variable.

Reply:
I am unfamiliar with this, could you point me to another place in the code with this?

Reply:
search for `get_property("sl_beta_test")` for examples

Reply:
you'll want something like `set_property("sl_forceTavern", true)` in the places where you currently set it true, and `set_property("sl_forceTavern", false)` in the right place alphabetically in `initializeSettings`.

Then just do `if(get_property("sl_forceTavern").to_boolean())` instead of `if(forceTavern)`